### PR TITLE
kubectl: Use Fprintf

### PR DESCRIPTION
**Pull request by gaurav1086**
Signed-off-by: Gaurav Singh <gaurav1086@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Use FprintF() instead of Fprintf() + Sprintf()
**Which issue(s) this PR fixes**:

Fixes #
N/A . Cleanup
**Special notes for your reviewer**:
A minor cleanup
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 1]
```diff
@@ -586,9 +586,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintln(o.ErrOut, fmt.Sprintf("No resources found in %s namespace.", o.Namespace))
+			fmt.FprintF(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:589]: `fmt.Fprintf` is the right function.

**Review by soltysh — CHANGES_REQUESTED** [review]
/assign

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go (hunk 1 above)
Comment (gaurav1086) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go]: Thanks. Fixed it.

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 2]
```diff
@@ -586,9 +586,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintln(o.ErrOut, fmt.Sprintf("No resources found in %s namespace.", o.Namespace))
+			fmt.Fprintf(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
 		} else {
-			fmt.Fprintln(o.ErrOut, fmt.Sprintf("No resources found"))
+			fmt.Fprintf(o.ErrOut, "No resources found\n")
```
Comment (neolit123) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:591]: should be `fmt.Fprintln(o.ErrOut, "No resources found")`

Comment (gaurav1086) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go]: Good point. Changes done. Thank you.

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 3]
```diff
@@ -586,9 +586,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintln(o.ErrOut, fmt.Sprintf("No resources found in %s namespace.", o.Namespace))
+			fmt.Fprintf(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
```
Comment (neolit123) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:589]: Better to use %q here?

Comment (gaurav1086) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go]: Yeah, I believe %q is better option. Changes done.

**Review by neolit123 — COMMENTED** [review]
please squash the commits to one.

then the PR needs approval from the kubectl maintainers.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
/retest

**Comment by liggitt** [comment]
/hold

this change is failing kubectl unit tests

File: staging/src/k8s.io/kubectl/pkg/cmd/get/get.go [hunk 4]
```diff
@@ -586,9 +586,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintln(o.ErrOut, fmt.Sprintf("No resources found in %s namespace.", o.Namespace))
+			fmt.Fprintf(o.ErrOut, "No resources found in %q namespace.\n", o.Namespace)
```
Comment (liggitt) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/get.go:589]: %s and %q are not equivalent and changes the output, failing unit tests

Comment (gaurav1086) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/get.go]: Reverted back to %s

**Pull request merged**